calico/networking/configuring/qos-controls.mdx

```diff
@@ -130,6 +134,20 @@ spec:
 |---------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `qos.projectcalico.org/dscp` | Specifies the DSCP value of egress traffic toward an endpoint outside cluser or another host in cluster.<br/>Min value: `0`<br/>Max value: `63`<br/>String values: DF, EF, AF11, AF12, AF13, AF21, AF22, AF23, AF31, AF32, AF33, AF41, AF42, AF43, CS0, CS1, CS2, CS3, CS4, CS5, CS6                                                                                                                                     |
 
+## Troubleshooting
+
+If limits for **bandwidth** on installations that use the **eBPF data plane** don't seem to be enforced correctly, check calico-node logs to see if `tcx` may not be supported by
+the operating system with `kubectl logs`. For example:
+
+```
+$ kubectl logs -n calico-system calico-node-cc7q2 | grep "tcx"
```

Comment:
Could you please check ```kubectl logs -n calico-system ds/calico-node``` works. IIRC, this should pick up the first pod in alphabetic order.

Reply:
It did work, thanks!